src/utils.js

```diff
@@ -41,6 +41,7 @@ function basicParser(results, ids, params = {}) {
   ids = ids.map(id => `${id}`);
   if (Array.isArray(results)) {
     return results.reduce((acc, curr) => {
+      if (!curr || !curr.id) return acc;
```

Comment:
Boyscout rule candidate.
Could be simplified by extracting the proper list of ids before we reduce.

```suggestion
      const resultIds = Array.isArray(results)
        ? ids.map(id => `${id}`)
        : Object.keys(results).map(id => `${id}`)
         
       return resultIds.reduce((accumulator, id)=> {
         if(ids.includes(id)){
           accumulator[id] = results[ids];
         }
         return accumulator;
       }, {});
```